content/projects/microbial/index.md

```diff
@@ -0,0 +1,137 @@
+| [ELIXIR Emerging Microbiome Community](https://elixir-europe.org/communities/marine-metagenomics) | | Bérénice Batut
+| µbioinfo Slack | |
+
+# microGalaxy Community
+
+<img class="img-fluid float-right" src="/projects/microbial/microgalaxy-logo.png" style="width:200px;" alt="Logo for microgalaxy"/>
+
+microGalaxy is a [community of practice](/community/practice) to
+* **Develop** and **Sustain** microbial data analysis in Galaxy
+* Implement **standardised "best practices"**
+* Expand **documentation** and **training**
+* **Coordinate efforts** to avoid redundancy in tools, workflows and training
+* **Share** the amazing things we are all doing together and outside Galaxy
+* Support each other
+
+
+## Meetings
+
+The microGalaxy community usually meets **4 times per year** (including 1 BoF at GCC and other community events) for a **2-hour meeting**. The meeting **time alternates** between 5-7pm Central European time and 9-11am Central European time in order to support members leaving in different timezones. The main aims of the meetings are to
+
+* Get news and updates from the community
+* Have 1 or more "short" talks about a project or a topic of interest
+* Discuss and coordinate our efforts.
+
+Details about the next events and minutes of previous meetings can be found [here](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit#).
+
+## Working groups
+
+2 working groups were started within the community to work together towards the goals of the commmunity:
+
+- A [**Tool working groups**](https://docs.google.com/document/d/14LJLyVgT29m63iovCww7nMDmfRAqOicedzxHu3aqXnE/edit?usp=sharing) with the aims to
+  - Create and maintain a **curated up-to-date list of the available tools in Galaxy ecosystem for microbial data analysis**
+  - Create a wishlist of tools to integrate in Galaxy
+  - **Assess the needs from users** for tools in Galaxy
+  - Organise **hackathons** to add tools in connection with the global community
+- A [**Paper(s) working groups**](https://docs.google.com/document/d/1uVQ8E_8HpIuHDYHwR82jU8pCvj0tnebFKyS0juAoyGc/edit?usp=sharing) with the aims to
+  - **Advertise about microbial data analysis** in Galaxy
+  - Investigate on the uses of Galaxy for microbial data analysis
+  - **Aggregate information about available resources and infrastructure** within Galaxy community
+  - Reflect on a **roadmap** for the community
+
+## Join us
+
+Anybody interested in microbial data analysis in the Galaxy is welcome to join microGalaxy! **Everybody is Welcome!**
+
+* Join our [**quarterly meetings**](https://docs.google.com/document/d/13VjcUjStuIp7bK29e74k8Nqb7N4lmVcg1ioArEWr254/edit#)!
+* Join our **working groups** (see above)
+* Join the [Gitter **Chat**](https://gitter.im/galaxyproject/microGalaxy) (also available via[ Matrix](https://usegalaxy-eu.github.io/index-microgalaxy.html?nonce=2775088#))
```

Comment:
This link does not lead to a matrix chat, https://usegalaxy-eu.github.io/index-microgalaxy.html?nonce=2775088#

Reply:
```suggestion
* Join the [Gitter **Chat**](https://gitter.im/galaxyproject/microGalaxy) (also available via[ Matrix](https://matrix.to/#/#galaxyproject_microGalaxy:gitter.im))
```